content/workshops/openshift_101_dcmetromap/lab3-s2i.md

```diff
@@ -18,8 +18,8 @@ We can do this either via the command line or the web console.  You decide which
 <i class="fa fa-terminal"></i> Goto the terminal and type the following:
 </blockquote>
 
-```
-$ oc new-app --name=dc-metro-map https://github.com/dudash/openshift-workshops.git --context-dir=dc-metro-map
+```bash
+$ oc new-app --name=dc-metro-map https://github.com/RedHatGov/openshift-workshops.git --context-dir=dc-metro-map
```

Comment:
I wanted to get all the code that we use over to RedHatGov because it looks more official. Maybe we can look at some kind of versioning scheme for the workshops or pulling from a particular branch. I'm open on that.